workflow/tests/compare.rb

```diff
@@ -26,10 +26,13 @@
       results[key]['rows'] = CSV.read(filepath)
     else
       puts "Could not find #{filepath}."
-      next
     end
   end
 
+  if (not results[base].keys.include?('rows')) || (not results[feature].keys.include?('rows'))
+    next
+  end
```

Comment:
@joseph-robertson What is this change for?

Reply:
@shorowit Basically, it allows handling the case where a feature branch adds or removes a results csv file.